.changeset/smart-grapes-double.md

```diff
@@ -0,0 +1,8 @@
+---
+"@wso2is/admin.rules.v1": minor
+"@wso2is/core": minor
```

Comment:
Shall we remove this as this PR doesn't contain changes to core module. Let's include admin.core.1 instead.

Reply:
There is a change to the core as well. https://github.com/wso2/identity-apps/pull/7454/files#diff-9645f12ad38068a9c5a082c00e09454d1c54bb1340bacad37eab41e14c880ef6

Reply:
My bad. I have bump the wrong module. I fixed it now.